src/features/import/store.ts

```diff
@@ -1,10 +1,11 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { RemoteList, remoteList } from 'utils/storeUtils';
 
 export interface ImportStoreSlice {
-  //TODO add interface
+  testList: RemoteList<{ id: string }>;
```

Comment:
No need for remote lists in this store since this feature handles no remote data (for the time being, at least).